.github/workflows/release_gems.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  workflow_dispatch:
+  push:
+    branch: [main]
+
+jobs:
+  source_gem:
+    name: Build+push source gem
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Workaround https://github.com/actions/checkout/issues/290
+        run: git fetch --force --tags
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+          bundler-cache: true
+      - name: "Install default uri gem version"
```

Comment:
This is added because uri version 0.10.0 is a default gem with Ruby 2.7, but net-http has uri 0.11.0 as a dependancy. https://github.com/cipherstash/activestash/runs/7832567913?check_suite_focus=true